snapcraft/plugins/cmake.py

```diff
@@ -64,22 +64,15 @@ def __init__(self, name, options, project):
         self.build_packages.append('cmake')
 
     def build(self):
-        if os.path.exists(self.builddir):
-            shutil.rmtree(self.builddir)
-        os.mkdir(self.builddir)
```

Comment:
I have no clue why we were deleting the builddir. A step before this had just populated it with a copy of the sources.